.azure-pipelines/recover_testbed/common.py

```diff
@@ -104,15 +104,29 @@ def posix_shell_onie(dut_console, mgmt_ip, image_url, is_nexus=False, is_nokia=F
                         dut_console.remote_conn.send("ip route add default via {}".format(gw_ip))
                         dut_console.remote_conn.send("\n")
 
-                        dut_console.remote_conn.send("onie-nos-install {}".format(image_url))
+                        # Remove the image if it already exists
+                        dut_console.remote_conn.send("rm -f {}".format(image_url.split("/")[-1]))
+                        dut_console.remote_conn.send("\n")
+
+                        dut_console.remote_conn.send("wget {}".format(image_url))
                         dut_console.remote_conn.send("\n")
                         # We will wait some time to connect to image server
                         time.sleep(60)
```

Comment:
better to remove the sleep outside of the for loop since there's already one in the for loop?